src/main/java/com/marklogic/kafka/connect/source/RowRecordConverter.java

```diff
@@ -0,0 +1,15 @@
+package com.marklogic.kafka.connect.source;
+
+import com.marklogic.client.row.RowRecord;
+import org.apache.kafka.connect.source.SourceRecord;
+
+/**
+ * Defines how a MarkLogic RowRecord is converted into a Kafka SourceRecord, which can then be
+ * sent to a Kafka Topic. Simplifies testing of this logic, as this avoids any dependency on a running MarkLogic
+ * or running Kafka instance.
+ */
+public interface RowRecordConverter {
```

Comment:
I think let's hold off on this as we're hoping this is a very simple operation - i.e. "new SourceRecord(topic, rowRecordAsJson)". Whereas on the "write" side, there's a lot of complexity to hide with converting a SinkRecord into a DocumentWriteOperation.